test-omni-integ-regtest.sh

```diff
@@ -9,7 +9,7 @@ trap cleanup EXIT
 BTCD=copied-artifacts/src/omnicored
 DATADIR=$HOME/.bitcoin
 LOGDIR=logs
-MSCLOG=/tmp/mastercore.log
+MSCLOG=/tmp/omnicore.log
```

Comment:
@dexX7 Is there still a log created in `/tmp`? Can we just remove that entirely. (It's used to create a symlink from $LOGDIR to a log file in /tmp.)